Model/ResourceModel/Currency.php

```diff
@@ -36,8 +36,39 @@ public function getByCurrency($code)
         return $connection->fetchAll($select);
     }
 
+    //should write  deletion routine too
+
+    /**
+     * @param $currencyCode string
+     * @return array
+     */
+    public function getPrecisionByCurrency($currencyCode)
```

Comment:
To retrieve precision info from the database. I think this comment should reside in the code itself.